ios/sdk/src/JitsiMeet.m

```diff
@@ -17,6 +17,7 @@
 #import <Intents/Intents.h>
 
 #import "Dropbox.h"
+#import "ExternalAPI.h"
```

Comment:
I don't think you need this import since the +Private header should already have it.